### DISCUSSION TITLE
Centering Icons after update?

### DISCUSSION THREAD

**Discussion by adlartuten** [AathifMahir/MauiIcons#82] [Q&A] [answered]
Hi, I have just updated this package but I am now having an issue where my icons are not centered.
![Screenshot 2024-01-31 144042](https://github.com/AathifMahir/MauiIcons/assets/25828555/70362b6d-038a-42bf-ab26-2b9125ecb918)
![Screenshot 2024-01-31 144102](https://github.com/AathifMahir/MauiIcons/assets/25828555/892f5a8d-13a0-4985-bc56-e8302f956d9f)

You can see the icon object is centered correctly in the circle but the icon itself is not centered. Am I doing something wrong here?

**Comment by AathifMahir** [comment]
> Hi, I have just updated this package but I am now having an issue where my icons are not centered. ![Screenshot 2024-01-31 144042](https://private-user-images.githubusercontent.com/25828555/301285989-70362b6d-038a-42bf-ab26-2b9125ecb918.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3NzQ0OTIsIm5iZiI6MTcwNjc3NDE5MiwicGF0aCI6Ii8yNTgyODU1NS8zMDEyODU5ODktNzAzNjJiNmQtMDM4YS00MmJmLWFiMjYtMmI5MTI1ZWNiOTE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAxVDA3NTYzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlNjNjZDhlZDEwNDUzYzk1MGIwOWM3Y2RhOTcxYzIwM2UzYzljZDkxM2E4ZjI2OWRjNDE5NzhmN2UyMmRkNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RvJDmxK9LE2HFJLaqGmUay-54VQJ8hunKRt2jed7LTA) ![Screenshot 2024-01-31 144102](https://private-user-images.githubusercontent.com/25828555/301286030-892f5a8d-13a0-4985-bc56-e8302f956d9f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3NzQ0OTIsIm5iZiI6MTcwNjc3NDE5MiwicGF0aCI6Ii8yNTgyODU1NS8zMDEyODYwMzAtODkyZjVhOGQtMTNhMC00OTg1LWJjNTYtZTgzMDJmOTU2ZDlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAxVDA3NTYzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjNjQwMTc5ZjMyMjdhNTk5MjVkYzk4OWUwMjgwYzhkMTdiNzI3ZTRmNjljNTcyMjQ5MTEyZDc2NzY3M2VjZjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qHxTcon5HJzA1yY82dJR4edvbYKWvJmka3t_wdbElP8)
> 
> You can see the icon object is centered correctly in the circle but the icon itself is not centered. Am I doing something wrong here?

I can't seem to reproduce this issue on Windows, May i know which platform your having the issue and also which version of MauiIcon that your using?

![image](https://github.com/AathifMahir/MauiIcons/assets/17165402/73496201-2c64-442c-a0a1-6e24c77ca3b3)

Here's the Xaml:

```xml
   <mi:MauiIcon
       BackgroundColor="White"
       HorizontalOptions="Center"
       Icon="{mi:Material Icon=Assignment}"
       IconColor="Black"
       IconSize="50" />
```

**Reply by adlartuten** [reply]
Sure thing, I'm using version 2.1.0 of MauiIcon and running my Maui app in Android .NET 8

**Reply by AathifMahir** [reply]
> Sure thing, I'm using version 2.1.0 of MauiIcon and running my Maui app in Android .NET 8

Let me try reproducing this issue on Android and get back to you, ASAP

**Reply by AathifMahir** [reply]
I'm able to reproduce this on Android

![image](https://github.com/AathifMahir/MauiIcons/assets/17165402/ba554796-3619-4dec-aaeb-728144bab35f)

Will diagnose and Provide you a solution as soon as possible

**Reply by AathifMahir** [reply]
I have released v2.1.1 with the Patch for this issue, Please update to Latest version and let me know, For now i'll close this discussion

**Comment by AathifMahir** [comment, marked as answer]
@adlartuten Does v2.1.1 fixes your issue?

**Reply by adlartuten** [reply]
Yep! They are correctly centered now
